src/app/dev/DevToys.Tools/Tools/Converters/Date/DateConverterCommandLineTool.cs

```diff
@@ -0,0 +1,162 @@
+﻿using System.Globalization;
+using DevToys.Tools.Helpers;
+using DevToys.Tools.Models;
+using Microsoft.Extensions.Logging;
+
+namespace DevToys.Tools.Tools.Converters.Date;
+
+[Export(typeof(ICommandLineTool))]
+[Name("DateConverter")]
+[CommandName(
+    Name = "dateConverter",
+    ResourceManagerBaseName = "DevToys.Tools.Tools.Converters.Date.DateConverter",
+    DescriptionResourceName = nameof(DateConverter.Description))]
+internal sealed class DateConverterCommandLineTool : ICommandLineTool
+{
+    [CommandLineOption(
+        Name = "number",
+        Alias = "n",
+        DescriptionResourceName = nameof(DateConverter.NumberOptionDescription))]
+    internal long? NumberOption { get; set; }
+
+    [CommandLineOption(
+        Name = "date",
+        Alias = "d",
+        DescriptionResourceName = nameof(DateConverter.DateOptionDescription))]
+    internal string? DateOption { get; set; }
```

Comment:
I am wondering if these two could be combined into one option of type `OneOf<long, string>` and make the option Required? Asking this because right now if I call `.\DevToys.CLI.exe dateconverter -tz test` I get no result (not even a message to tell that I'm missing an input, and the `tz` is invalid).